### PR TITLE
Lesson 1

### PR DESCRIPTION
**Pull request by yanyshyn**
Completed home work for lesson 1

### CODE REVIEW

File: lesson_1/Task1.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+if __name__ == '__main__':
+    Summ = 3**128*2**128
+print('Expression equals:', Summ)
```
Comment (itea-artem) [inline lesson_1/Task1.py]: No new line at the end of file. Add new line at the end of file everywhere.

File: lesson_1/Task3.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+if __name__ == '__main__':
+    String: str = "Напишите программу, которая считает сколько букв ‘o’ в заданной строке"
```
Comment (itea-artem) [inline lesson_1/Task3.py]: You can't name variable as data types (str, dict, list, tuple, set, int)

File: lesson_1/Task5.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
+if __name__ == '__main__':
+    First = str(input('Input first number: '))
```
Comment (itea-artem) [inline lesson_1/Task5.py]: PEP8: file name and variable name.

File: lesson_1/Task7.py [hunk 4]
```diff
@@ -0,0 +1,4 @@
+if __name__ == '__main__':
+    String = 'Напишите,программу, которая, преобразует, строку, в, которой ,записаны ,слова, через, “,”, в, список, из, этих, слов'
```
Comment (itea-artem) [inline lesson_1/Task7.py]: PEP8

File: lesson_1/Task1.py (hunk 1 above)
Comment (yanyshyn) [reply lesson_1/Task1.py]: fixed

File: lesson_1/Task3.py (hunk 2 above)
Comment (yanyshyn) [reply lesson_1/Task3.py]: fixed

File: lesson_1/Task5.py (hunk 3 above)
Comment (yanyshyn) [reply lesson_1/Task5.py]: fixed

File: lesson_1/Task7.py (hunk 4 above)
Comment (yanyshyn) [reply lesson_1/Task7.py]: fixed

**Pull request closed without merging**